src/mocap_node.cpp

```diff
@@ -52,7 +52,9 @@ class OptiTrackRosBridge
                      PublisherConfigurations const& pubConfigs) :
     nh(nh)
```

Comment:
why not use the initializer list? Putting this in a shared pointer is not necessary